### ISSUE TITLE
[mesheryctl] Replace logrus with meshkit logger

### ISSUE THREAD

**Issue by piyushsingariya** [meshery/meshery#4705]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

mesheryctl is using https://github.com/sirupsen/logrus everywhere

#### Desired Behavior

`Logrus` needs to be replaced with `meshkit` logger

---

#### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by ayanchoudhary** [comment]
Hi, can I work on this issue if this is free?

**Comment by piyushsingariya** [comment]
> Hi, can I work on this issue if this is free?

Yeah, sure! btw it would better if this issue is tackled in small multiple PRs

**Comment by ayanchoudhary** [comment]
just a clarification, this replacement needs to be done only for mesheryctl and should leave everything else as it is, am I correct?

**Comment by ayanchoudhary** [comment]
and the PRs should be broken down based on different log-levels?

**Comment by piyushsingariya** [comment]
> just a clarification, this replacement needs to be done only for mesheryctl and should leave everything else as it is, am I correct?

yes in each folder under mesheryctl.

> and the PRs should be broken down based on different log-levels?

Nope, just enough changes that Maintainers doesn't get overwhelmed during reviewing PRs.

**Comment by leecalcote** [comment]
Issue assignment seems to have gone stale, so freeing up for others to contribute. @ayanchoudhary, just signal if that isn't the case.

**Comment by piyushsingariya** [comment]
Closed via pull request #5085

**Comment by deepto98** [comment]
@leecalcote From what I understand, the meshkit logger is already implemented, but logrus is still being used and needs to be removed completely?

**Comment by leecalcote** [comment]
Yes

**Comment by deepto98** [comment]
@leecalcote Okay, I'm going through the prevoius replacement PRs, trying to understand the meshkit logger usage, will create a PR when its done

**Comment by leecalcote** [comment]
There is already a PR on this.

**Comment by Yorisoft** [comment]
@lekaf974 Not sure how to manage it but no single PR should resolve this issue. Perhaps maybe the last PR but these first three are not that.  Would it be alright to reopen?
